e2e_tests/tests/cluster/utils.py

```diff
@@ -100,10 +100,13 @@ def get_command_info(command_id: str) -> Dict[str, Any]:
     return get_task_info("command", command_id)
 
 
-def command_succeeded(command_id: str) -> bool:
-    print(get_command_info(command_id))
-
-    return "success" in get_command_info(command_id)["exitStatus"]
+# assert_command_succeded checks if a command succeeded or not. It prints the command info in case
+# of a failure.
+def assert_command_succeeded(command_id: str) -> None:
+    command_info = get_command_info(command_id)
+    command_info_json = json.dumps(command_info, indent=2, separators=(",", ":"))
+    succeeded = "success" in command_info["exitStatus"]
+    assert succeeded, f"Command failed. Command Info:\n {command_info_json}"
```

Comment:
This doesn't seem much better.   We just basically get a non-zero status message, and not the actual error messages.

I think you should use something more like what 

def report_failed_experiment(experiment_id: int) -> None:

does, and display the task_logs.

Reply:
Sure, Jerry. Updated as suggested.